Solutions/Corvus.SpecFlow.Extensions/Corvus/SpecFlow/Extensions/FunctionsController.cs

```diff
@@ -32,13 +27,18 @@ namespace Corvus.SpecFlow.Extensions
     /// </remarks>
     public sealed class FunctionsController
```

Comment:
I think we should avoid the ugliness by renaming this class - I'd suggest we go for `FunctionsControllerBindings` and modify the way we expect it to work with the SpecFlow `ScenarioContext` and `FeatureContext` to bring it in line with the way `ContainerBindings` works. I'm happy to do this bit myself if you like?

Reply:
Yeah, on this one some guidance at the very least would be appreciated. I'm not sufficiently familiar with SpecFlow to understand what you have in mind for the end point of that refactoring.

Reply:
I was getting rid of this class as you typed...